keadm/hack/update-codegen.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# Copyright 2020 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBEEDGE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
+TMP_DIR="${KUBEEDGE_ROOT}/_tmp"
+
+SUPPORT_VERSION="1.3
+1.4
+1.5"
```

Comment:
Great solution, small question need to be care:
The code generating here rely on the release branch, If we release a new version, we will first create a new branch, and keadm still have been updated to new version...

Reply:
And the support version we can get from the website instead of hard code here?

Reply:
Now we provide tarballs after create a release branch, so it may be no problem.

I think we can update this when releasing.